src/evaluation_system/api/parameters.py

```diff
@@ -32,47 +32,47 @@
-         always the option to provide multiple values by re-using the same parameter
-         name (e.g. ``param1=a param1=b`` produces ``{'param1': ['a', 'b']}``). But the
-         configuration file does not allow this at this time. Therefore is better
-         to setup a separator, even though the user might not use it while giving
-         input. It must not be a character, it can be any string
-         (make sure it's not a valid value!!)
-     regex: Optional[str], default: None
-         A regular expression defining valid "string" values before
-         parsing them to their defining classes (e.g. an Integer might define a
-         regex of "[0-9]+" to prevent getting negative numbers). This will be used
-         also on Javascript so don't use fancy expressions or make sure they are
-         understood by both python and Javascript.
-     help: str, default: No help available
-         The help string describing what this parameter is good for.
-     print_format: str, default %s
-         String format used to display parameter values, e.g. ``%.2f`` to
-         display always 2 decimals for floats
-     impact:
-         The impact of the parameter to the output, possible values are
-         Parameter.Impact.affects_values, Parameter.Impact.affects_plots, Parameter.
-         Impact.no_effects
+        boolean indicating if this parameter is required
+    max_items: int, default: 1
+        If set to > 1 it will cause the values to be returned in a
+        list (even if the user only provided 1). Raises an error if more than
+        than ``max_items`` values are parsed.
+    item_separator: str, default: ,
+        The string used to separate multiple values for this
+        parameter. In some cases (at the shell, web interface, etc) the user have
+        always the option to provide multiple values by re-using the same parameter
+        name (e.g. ``param1=a param1=b`` produces ``{'param1': ['a', 'b']}``). But the
+        configuration file does not allow this at this time. Therefore is better
+        to setup a separator, even though the user might not use it while giving
+        input. It must not be a character, it can be any string
+        (make sure it's not a valid value!!)
+    regex: Optional[str], default: None
+        A regular expression defining valid "string" values before
+        parsing them to their defining classes (e.g. an Integer might define a
+        regex of "[0-9]+" to prevent getting negative numbers). This will be used
+        also on Javascript so don't use fancy expressions or make sure they are
+        understood by both python and Javascript.
+    help: str, default: No help available
+        The help string describing what this parameter is good for.
+    print_format: str, default %s
+        String format used to display parameter values, e.g. ``%.2f`` to
+        display always 2 decimals for floats
+    impact: int
```

Comment:
This is not really an integer. To be honest I don't even understand this parameter at all.

Reply:
I was looking at models.py and the last line showed 

```
    impact = models.IntegerField(choices=IMPACT_CHOICES, default=Impact.affects_values)
```
and my vscode pluggin was autosuggesting that. That is why I put it (and because when I was building the html I had a very weird indentation if I did not put sthing besides the parameter), but yes, fair enough. I can take it back.